docs/run-api-server/installing.mdx

```diff
@@ -28,6 +29,40 @@ sudo apt install stellar-horizon stellar-core
 
 </CodeExample>
 
+## Helm Chart Installation
+
+If the deployment can be done on Kubernetes, there is a [horizon helm chart](https://github.com/stellar/helm-charts/blob/main/charts/horizon) available.
+
+<CodeExample>
+
+```bash
+$ git clone https://github.com/stellar/helm-charts; cd helm-charts
```

Comment:
Maybe we can install straight from the helm repo? Something like this might work (untested):
```
helm repo add stellar https://helm.stellar.org/charts && helm repo update stellar
helm install my-horizon stellar/horizon --namespace my-horizon-namespace-on-cluster --set global.image.horizon.tag=XXX 
```

Reply:
ok, I'll test the chart deploys for horizon/core direct from repo on sandbox. For setting values on helm, which do you think we should recommend for admins to use from helm cli, get a local copy of the `values.yml` from the charts repo, or recommend `--set`s on specific keys here? I was leaning towards values.yml, since it scales a bit better over time, as it won't become stale, it could be linked to the values.yml in charts repo, rather than documenting  multiple `--set`s here which could require maintenance to keep up to date.

Reply:
@jacekn , found an issue during helm install from repo on horizon chart - https://github.com/stellar/helm-charts/issues/64

Reply:
> For setting values on helm, which do you think we should recommend for admins to use from helm cli, get a local copy of the values.yml from the charts repo, or recommend --sets on specific keys here? 

If all we need to set is the image tag I'd say `--set` is nicer because it's short and concise. Maybe we can use that and mention the values method for people who need to customize things?

Reply:
it might suffice to use just `--set` for most of use cases, and include one more for `global.network`, I'll mention to checkout out values.yml from repo for any non standard use case, i.e. if the user wants to use any network other than testnet/pubnet, then they have to manually set many other attributes in values.yml.